back/config/locales/sr-SP.yml

```diff
@@ -66,7 +66,7 @@ sr-SP:
+    infopage_body: "<p> Ова <strong> платформа за учешће </strong> иницијатива је ваше општине или организације и може се користити за различите процесе јавних консултација: прикупљање идеја, гласање, дискусије, анкете, упитнике, партиципативно буџетирање, мапирање, планирање сценарија, подршку саветодавним одборима ... </p> \n<p> На почетној страници видећете који <strong> су процеси активни </strong> и у којој су <strong> фази </strong> тренутно. </p> \n<p><img data-cl2-text-image-text-reference=\"e2c7bc7a-017d-4887-a3cb-b94185617a59\" alt=\"Horizontal separation rule\" data-align=\"center\" style=\"display: block; margin: auto;\"></p>\n<h2> Како да се пријавим на платформу? </h2>\n<p> Пријављивање се врши лако - навођењем имена и презимена и емаил адресе. Такође, можете да користите свој Гоогле или Фацебоок налог. Само ће ваше име и презиме бити видљиво другима. </p> \n<h2> Могу ли сам да користим платформу, нпр. како бих организовао јавну расправу? </h2>\n<p> Апсолутно. Затражите од одговорних у вашој општини или организацији да вам дају потребна права и дозволе како бисте постали модератор пројекта. Једном када добијете ове дозволе, можете да поставите свој пројекат и временске оквире, као и да делите информације које желите (видео, документе, слике ...). Ово ће вам такође омогућити да позовете друге на платформу, и подстакнете их да постављају идеје у оквиру вашег пројекта, гласају или коментаришу. </p> \n<h2> Могу ли да учествујем као удружење/организација/одбор? </h2>\n<p> Да, то је могуће. С обзиром на то да су другима само ваша имена и презимена видљива на платформи, можете једноставно користити име своје организације приликом креирања профила. </p> \n<h2> Зашто бих користио ову платформу, уместо да сам креирам своју wеб страницу? </h2>\n<p> Тим посвећених програмера и стручњака за онлајн учешће <a href=\"https://www.citizenlab.co/en\" target=\"_blank\">CitizenLab,</a> по налогу ваше општине или организације) свакодневно побољшава и ажурира ову платформу. Чак се и најмања грешка решава одмах, а ако имате креативне предлоге, они вам могу помоћи да увидите које се функције платформе могу користити за њих. <p><img data-cl2-text-image-text-reference=\"392d0e47-e5f9-41ab-9ceb-affac617b8b1\" alt=\"Horizontal separation rule\" data-align=\"center\" style=\"display: block; margin: auto;\"></p>\n<п><strong> <em> Потребно вам је више информација? </em> </strong> <em> Желите ли да користите платформу? Питања или утисци? Обратите се одговорнима у вашој општини или организацији, или контактирајте ЦитизенЛаб путем суппорт@цитизенлаб.цо. </em> </p> <strong>  "
```

Comment:
Corrects an invalid `<img data-cl2-text-image-text-reference=\"e2c7bc7a-017đ4887-a3cb-b94185617a59\"` to `<img data-cl2-text-image-text-reference=\"e2c7bc7a-017d-4887-a3cb-b94185617a59\"`

See [CL-1649](https://citizenlab.atlassian.net/browse/CL-1649) for details.